docs/build-provenance-for-all-registries.md

```diff
@@ -0,0 +1,104 @@
+# Build Provenance for All Registries
+
+There’s a new security capability that registries can offer - build provenance - which verifiably links a package back to its source code and build instructions.
```

Comment:
I think there's room for a *touch* more motivation:

> There’s a new security capability that registries can offer - build provenance - which verifiably links a package back to its source code and build instructions. When package source is publicly available, this prevents an attacker from building a different, malicious artifact and publishing that to the registry.

Reply:
You do hit on this below but I think getting it in the first sentence is nice too

Reply:
I have added additional text.